scripts/setup.js

```diff
@@ -0,0 +1,104 @@
+const fs = require('fs')
+const path = require('path')
+
+/* 
+****************************
+Update package.json
+****************************
+*/
+
+const packageJSON = require('../package.json')
+const { name: projectName } = packageJSON
+
+// ensure that yarn init has been run
+if (projectName === undefined) {
+  console.log(
+    'Warning: no name defined in package.json. Please run "yarn init" before "yarn setup".',
+  )
+  process.exit(1)
+}
+
+// 1. Change "name" in the "jbrowse-plugin" field to the name of your project (e.g. "MyProject")
+packageJSON['jbrowse-plugin'].name = projectName
```

Comment:
Generally the `jbrowse-plugin` name is different from the package name. It should have any `@scope` and the "jbrowse-plugin-" stripped from it and be converted to PascalCase. This is also what should be used in the start and build scripts and the "name" of the plugin in any config file.

Reply:
Fixed